src/ListItem/index.tsx

```diff
@@ -183,7 +194,7 @@ export const ListItem = React.forwardRef<any, React.PropsWithChildren<Props>>(
                   })
                 )
               )}
-              ref={ref}
+              ref={ref as React.Ref<HTMLDivElement>}
```

Comment:
Unfortunately using `HTMLElement` doesn't work:

>Type 'HTMLElement | null' is not assignable to type 'HTMLDivElement | null'.

I can't assign a ref of `HTMLElement` to `HTMLDivElement` because `HTMLElement` doesn't satisfy the requirement of having all the props that `HTMLDivElement` is capable of. I _believe_ going the opposite direction would work because extra properties are ok, just not a lack.

Reply:
I'm thinking the forwardRef should be HTMLDivElement? Or does it need to be HTMLElement for another reason?
```
export const ListItem = React.forwardRef<
  HTMLDivElement,
  React.PropsWithChildren<Props>
>(
```

Reply:
Great point; I was trying to hide the internal implementation but there isn't really a point to that since we're forwarding the ref. I'll change it!